.travis.yml

```diff
@@ -4,10 +4,6 @@ matrix:
     fast_finish: true
 perl:
    - '5.24'
-   - '5.22'
```

Comment:
why all these versions are removed?

Reply:
Since we don't use them internally and they're not officially supported, it seems wasteful to have travis trigger too many jobs here.